.eslintrc.json

```diff
@@ -0,0 +1,81 @@
+{
+  "root": true,
+  "ignorePatterns": [
+    "**/dist/**/*",
+    "**/node_modules/**/*"
+  ],
+  "overrides": [
+    {
+      "files": [
+        "*.ts",
+        ".js"
+      ],
+      "rules": {
+        "arrow-body-style": "error",
+        "brace-style": [
+          "error",
+          "1tbs",
+          { "allowSingleLine": true }
+        ],
+        "constructor-super": "error",
+        "curly": "error",
+        "dot-notation": "off",
```

Comment:
We should enable this.

`field.property` is preferable to `field["property"]`.

Reply:
not always